### DISCUSSION TITLE
Just how planned is OAuth really?

### DISCUSSION THREAD

**Discussion by CEbbinghaus** [jellyfin/jellyfin#7720] [General]
So Authelia is finally receiving compatibility with Caddy allowing me for the first time to use SSO for any of my internal apps I use. To this end I did some digging and although I have found many threads asking for OAuth support I haven't found any concrete work being done on it. So I was wondering what the progress is and if the feature is planned at all (for the near future). I would be up to try and contribute it myself however I am extremely busy and have failed to try to delve into the codebase a few times already.

Hoping this will be added soon since it would make for a great way to simplify the sign-in process.

**Comment by crobibero** [comment]
https://github.com/9p4/jellyfin-plugin-sso

**Reply by CEbbinghaus** [reply]
It's a start but only has support for Web which is nowhere enough. What I want to know is if the rework of the auth system with OAuth support is planned.

Note djbon2112 noted [here](https://www.reddit.com/r/jellyfin/comments/g3etp0/comment/fnr6go5/?utm_source=share&utm_medium=web2x&context=3) that the current auth codebase is a mess and needs reworking

**Comment by ServusNL** [comment]
Not really planned clearly :( Looking forward to this enhancement but all issues, feature pages etc are almost abandoned.
